src/main/java/com/example/shorturl/security/SecurityConfiguration.java

```diff
@@ -38,7 +43,8 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception{
                 .exceptionHandling().authenticationEntryPoint(jwtAuthEntryPoint).and()
                 .sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS).and()
                 .authorizeRequests()
-                .antMatchers("/api/**").permitAll()
+                .antMatchers("/api/v1/**").permitAll()
+                .antMatchers("/api/v2/**").hasAnyRole("ADMIN", "USER")
                 .antMatchers("/admin/**").hasRole("ADMIN")
```

Comment:
What is the difference between this and line 50?

Reply:
I thought that "/admin/**" was not considering "/admin". I checked and it is the same. Let me erase that line